app/controllers/api/v1/diaper_calculators_controller.rb

```diff
@@ -36,6 +36,14 @@ def diapers_service_handler(age)
       def childs_age
         params[:childs_age].to_i
       end
+
+      def product_price
+        diaper = Product.find_by title: 'diaper'
+        selected_price = ProductPrice.find_by category: params[:price_id],
```

Comment:
()

Reply:
from where do you get params[:price_id]?
what if this parameter is empty? (when the feature flag is turned off)

Reply:
Okay. I created test for this. All done. Default value returned